sips/voluntary_exit.md

```diff
@@ -0,0 +1,247 @@
+| Author      | Title                          | Category | Status |
+|-------------|--------------------------------|----------|--------|
+| Alon Muroch | Voluntary Exit | Core     | Draft  |
+
+
+**Summary**
+Describes how zero coordination voluntary exit works.  
+<em>Important to note that a user holding the validator’s private key can sign a voluntary exit as well.</em>
+
+
+**Specification**
+
+**Contract**  
+Adds a new function accessible only by the validator's owner, pushes a voluntray exit event. 
+Can only be called once.
```

Comment:
@alonmuroch Why only once?

Reply:
Could be multiple times

Reply:
Updating to multiple times.

Reply:
@MatheusFranco99 just confirming it could be multiple times. thanks.

Reply:
Thanks @mtabasco